src/functions-and-arrays.js

```diff
@@ -1,41 +1,165 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
+//console.log(maxOfTwoNumbers(4, 5));
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
-
+function findLongestWord(arr) {
+  let longestWord = '';
+  for (let i = 0; i < arr.length; i++) {
+    //console.log(arr[i]);
+    //console.log(arr[i].length);
+    if (arr[i].length > longestWord.length) {
+      //console.log(arr[i]);
+      longestWord = arr[i];
+      // console.log(arr[i]);
+    }
+    //console.log(longestWord);
+  }
+  if (arr.length === 0) {
+    return null;
+  } else {
+    return longestWord;
+  }
+}
+console.log(findLongestWord([]));
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
-
-function sumNumbers() {}
-
-
+/*
+function sumNumbers(arr) {
+  let sum = 0;
+  for (let i = 0; i < arr.length; i++) {
+    //console.log(arr[i]);
+    sum += arr[i];
+  }
+  return sum;
+}*/
+
+function sumNumbers(arr) {
+  let sum = 0;
+  for (let el of arr) {
+    if (typeof el === 'string') {
+      sum += el;
+    } else {
+      sum += el;
+    }
+  }
+  return sum;
+}
```

Comment:
You are missing one test on this exercise. Try and research the "Throw Error" phrase for a better understanding of how it works